src/Enchantments.h

```diff
@@ -103,6 +104,9 @@ class cEnchantments
 	/** Returns true if there are no enchantments */
 	bool IsEmpty(void) const;
 
+	/** Returns true if all of our enchantments are compatible with the specified enchantment */
+	bool IsCompatibleWith(int a_EnchantmentID) const;
```

Comment:
Also, what happens if specifying an enchantment already present? Should it return true or false? Doc.